cypress/e2e/Welcome_checks.js

```diff
@@ -0,0 +1,140 @@
+import {sign_in_page} from "../selectors/sign_in_page_selectors";
+import {main_page} from "../selectors/main_page_selectors";
+import {sign_up_page} from "../selectors/sign_up_page_selectors";
+
+
+const uuid = () => Cypress._.random(111111111, 99999999)
+const id = uuid()
+const randomNumber = `${id}`
+
+
+describe('Checks of the artifacts on a page', () => {
+
+  before(() => {
+    cy.visit('/')
+  })
+
+  //MINOR CHECKS
+  it('should show Username and Password placeholders', () => {
+    cy.get(sign_in_page.username_placeholder).should('be.visible')
+    cy.get(sign_in_page.password_placeholder).should('be.visible')
+  })
+
+  it('should show typeable Username field', () => {
+    cy.get(sign_in_page.login_field).type('IhorHolubtsov')
+    cy.get(sign_in_page.login_field).should('have.value', 'IhorHolubtsov')
+    cy.get(sign_in_page.login_field).clear()
+  })
+
+  it('should show typeable Password field', () => {
+    cy.get(sign_in_page.password_field).type('RestTest1!')
+    cy.get(sign_in_page.password_field).should('have.value', 'RestTest1!')
+    cy.get(sign_in_page.password_field).clear()
+  })
+
+  it('should show \'Username is required\' error if user clicks on it and then click outside this field and didn\'t enter any value', () => {
+    cy.get(sign_in_page.password_field).click()
+    cy.get(sign_in_page.username_error_message).should('be.visible')
+  })
+
+  it('check "Remember me" checkbox', () => {
+    cy.get(sign_in_page.remember_me_checkbox).click().should('be.checked').click().should('be.not.checked')
+  })
+
+  it('should show disabled by default sign in btn', () => {
+    cy.get(sign_in_page.login_field).should('have.value', '')
+    cy.get(sign_in_page.sign_in_button).should('be.disabled')
+  })
+
+  it('should have \'Don\'t have an account? Sign Up\' clickable link under \'Sign in\' btn', () => {
```

Comment:
You can use double quotes, in this case, there is no need to use the escape character

Reply:
fixed